orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/DeploymentDetailsAware.groovy

```diff
@@ -35,8 +35,11 @@ trait DeploymentDetailsAware {
     def result = [:]
     if (previousStage) {
       if (previousStage.context.containsKey("amiDetails")) {
-        result.amiName = previousStage.context.amiDetails.ami
```

Comment:
Not sure how this ever worked previously ... amiDetails is a `List<Map>` and `amiDetails.ami` would have resolved to a `List<String>`.